package.json

```diff
@@ -161,6 +161,7 @@
     "fs-extra": "^10.0.1",
     "graphql": "15.8.0",
     "graphqurl": "^1.0.1",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
Our Jest config was setup to use this plugin but we didn't actually have it as a dependency. It just so happened that we have so few tests that none of them had hit this path before I added a CSS module in the new `ConsoleSearch` component.